src/FilterPlugin.php

```diff
@@ -107,3 +89,4 @@ public function sendPerformed(Swift_Events_SendEvent $event)
 		// Do Nothing
 	}
 }
+
```

Comment:
Could you remove the extra blank line at the end? A file should have a newline character at the end, but not have an extra empty line.

That's true for all the other new files as well.

Reply:
Hah man sorry for the tangent, it's been so long since I've worked with PHP. In typescript land land we have `prettier` to automatically solve those kinds of style adherence problems, isn't there something like this for PHP already?

Reply:
There is, of course, we just haven't set it up through the all the repos. We use [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) and [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer).